object/check_password_complexity.go

```diff
@@ -52,11 +52,13 @@ func isValidOption_Aa123(password string) string {
 	return ""
 }
 
-func isValidOption_SpecialChar(password string) string {
-	if !regexSpecial.MatchString(password) {
-		return "The password must contain at least one special character"
+func isValidOption_SpecialChar(password string, specialChars string) string {
+	for _, p := range password {
+		if strings.ContainsRune(specialChars, p) {
+			return ""
+		}
 	}
-	return ""
+	return "The password must contain at least one special character"
```

Comment:
it is possible, but since the size of the line is not limited, there is a possibility that there will be too many characters

Reply:
It make sense. If we will truncate line thou in can fit interests for most users. 
In case long string the message can have ellipsis and word etc. 

It is just a suggestion, in original implementation with hard-coded symbols there are no such details. So OK.